### DISCUSSION TITLE
Detect LED on/off

### DISCUSSION THREAD

**Discussion by tompsg-git** [jomjol/AI-on-the-edge-device#843] [Q&A]
Hallo,
ich habe auf meinem Solarregler einige LEDs, die den Zustand anzeigen. Ist es möglich die LEDs als on oder off zu erkennen. Ggf. wäre eine Farberkennung ebenfalls sinnvoll.
Viele Grüße,
Thoma
![gsobild_5157](https://user-images.githubusercontent.com/98977001/176763546-076c4fd7-5785-405d-ba85-d750733ade18.jpg)
s

**Comment by jomjol** [comment]
Der technische Background ist dafür geeignet, allerdings würde dies umfangreichere Anpassungen im Code und ein gesondertes Training erfordern. Aufwand ist für eine einzelne Anwendung ziemlich hoch.

**Reply by RMadosa** [reply]
Hallo jomjol

Genau diese Frage wie oben gestellt ging mir auch durch den Kopf, als ich mal meine Heizung- bzw. Solarsteuerungseinheiten angeschaut habe.

Ich kenne mich mit KI nicht so wirklich aus, doch ist denn eine Zahl nicht vergleichbar mit einem Punkt?
Wenn es nur um die Zeit geht des Antrainieren geht, würde ich hier meine Hilfe anbieten. Gebe aber zu, das ich noch keine Ahnung von habe.

Gut ein eigener ESP könnte das ja ggf mit einer Fotodiode dann in eine eigene 7 Segment-Anzeige "umschlüssel" oder direkt an den ioBrocker schicken und ja dann wäre es auch möglich eine/mehrere DS18S20 für die Temperatur (Hin-/Rückweg) mit einzubringen. Das ist ja dann doch wieder ein zusätzlicher "Stromfresser" mehr und auch ein ganz anderes Thema in Bezug auf die Frage. :-)

Sorry, wenn die Annahme helfen zu können (ohne ausreichendem Grundverständis) bzw. die Frage ansich dann doch zu naiv ist.

Aber biete mich gerne als Beta-Tester an.
